modules/installation-aws-config-yaml.adoc

```diff
@@ -362,7 +358,7 @@ endif::openshift-origin[]
 endif::private[]
 ifdef::secret[]
 ifndef::openshift-origin[]
-<14> The custom CA certificate. This is required when deploying to the AWS C2S Top Secret Region because the AWS API requires a custom CA trust bundle.
+<14> The custom CA certificate. This is required when deploying to either the SC2S or Commercial Cloud Services (C2S) Regions because the AWS API requires a custom CA trust bundle.
```

Comment:
> SC2S or Commercial Cloud Services (C2S) Regions
It is better to be consistent
* SC2S or C2S Regions
or 
* Secret Commercial Cloud Services (SC2S) or Commercial Cloud Services (C2S) Regions

Reply:
Agreed. Thank you.